app/preload/database/transactionsDao.ts

```diff
@@ -34,6 +34,15 @@ async function getPendingTransactions(): Promise<TransferTransaction[]> {
     return transactions;
 }
 
+async function getTransaction(id: string): Promise<TransferTransaction> {
```

Comment:
This is typed incorrectly as it will return `undefined` if there are no results. You can add the typing by adding:
```
.table<TransferTransaction>(transactionTable)
```
as it will then return `TransferTransaction | undefined`. It can also be used `getTransactionsOfAccount`.